src/main/java/org/opentripplanner/model/fare/FareProduct.java

```diff
@@ -47,10 +47,10 @@ public String toString() {
     var builder = ToStringBuilder
       .of(FareProduct.class)
       .addStr("id", id.toString())
-      .addObj("amount", price);
-    builder.addDuration("duration", validity);
-    builder.addObj("category", category);
-    builder.addObj("medium", medium);
+      .addObj("amount", price)
+      .addDuration("duration", validity)
+      .addObj("category", category)
```

Comment:
Not a big deal since this is just the toString, but should this be `riderCategory`?

Reply:
The type of the field is indeed `RiderCategory` but its name is `category`, so the toString is correct.

That doesn't mean that the variable name cannot be changed.